Cesium3DTilesSelection/test/TestTilesetSelectionAlgorithm.cpp

```diff
@@ -1554,21 +1560,30 @@ void runUnconditionallyRefinedTestCase(const TilesetOptions& options) {
       pRawLoader->createRootTile(),
       options);
 
-  // On the first update, we should render the root tile, even though nothing is
-  // loaded yet.
+  // On the first update, we should refine right to the grandchild tile, even
```

Comment:
The nitpickiest of nitpicks...

```suggestion
  // On the first update, we should refine down to the grandchild tile, even
```